README.md

```diff
@@ -7,24 +7,29 @@ Elastic AI implementation for the Elastic Node version 5 (enV5).
 
 ## Setup
 
-For compiling the project the tools are:
+For compiling the project, the required tools are:
 
-- required:
-  - gcc (local C compiler)
-  - arm-non-eabi-gcc (C compiler for Target device)
-  - CMake (Build Script Generator)
-  - Ninja (Build Tool)
-- recommended:
-  - pre-commit
+- gcc (local C compiler)
+- arm-non-eabi-gcc (C compiler for RP2040)
+- CMake (Build Script Generator)
+- Ninja (Build Tool)
+
+We recommended pre-commit to check that the commit message align with the conventional commit mesage standard.
```

Comment:
The pre-commit stuff is in fact outdated. I didn't fixed anything regarding the commitlint checks, because to my knowledge that stuff is part of Issue https://github.com/es-ude/HiWi-s-Guide/issues/1 and should therefore be fixed by Jasmin. I don't know how far her stuff is done. I would resolve such issues with her by finishing her Issues at next.